docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -6,13 +6,55 @@
    "source": [
     "# Quantum Kernel Machine Learning\n",
     "\n",
-    "The general task of machine learning is to find and study patterns in data. For many datasets, the datapoints are better understood in a higher dimensional feature space, through the use of a kernel function:\n",
+    "## Overview\n",
+    "\n",
+    "Machine learning and quantum computing are two technologies each with the potential for altering how computation is performed to address previously unsolvable problems. Kernel methods for machine learning are good for pattern recognition, with support vector machines (SVMs) being the most well-known method for classification problems.\n",
+    "\n",
+    "The tutorial is structured as follows:\n",
+    "1. [Classification](#Classification) \n",
+    "    * Classification using classical SVC using quantum kernel\n",
+    "    * Classification using quantum QSVC\n",
+    "    \n",
+    "    \n",
+    "2. [Clustering](#Clustering)\n",
+    "    * Clustering using classical Spectral algorithm using quantum kernel\n",
+    "    \n",
+    "This tutorial is based on the following research paper: \n",
+    "\n",
+    "[_Supervised learning with quantum enhanced feature spaces_](https://arxiv.org/pdf/1804.11326.pdf), this can be referred for more information.\n",
+    "\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "#### Classical Perspective:\n",
+    "\n",
+    "Support Vector Machine (SVM) is a machine learning algorithm that is used for classification, regression and clustering purposes. Generally, SVMs can be used for linear classification purposes. Additionally, they can efficiently perform a non-linear classification using the kernel trick. The kernel trick enables the mapping of inputs into high dimensional feature spaces.\n",
```

Comment:
I don't think regression is a typical use case for SVMs.

```suggestion
    "Support Vector Machines (SVMs) are machine learning algorithms used for classification and clustering purposes. While SVMs can be used for linear classification, their power relies on being able to efficiently perform non-linear classification using the kernel trick.\n",
```

Reply:
In a college course, I was taught implementation of SVM on regression and classification problem. But as we are exploring classification and clustering in the tutorial, so we can mention the use accordingly